site/content/en/guides/consumer/function/_index.md

```diff
@@ -10,16 +10,19 @@ description: >
 ## Functions Explained
 
 KRM Config Functions are client-side programs that make it easy to operate on a
```

Comment:
Since we introduce them as KRM Config functions here, let's remove any mentions of kpt functions at least in this page.

Reply:
Done.